src/plugins/enum/README.md

```diff
@@ -0,0 +1,19 @@
+- infos = Information about the enum plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/placements = presetstorage
+- infos/description =
+
+## Introduction ##
+
+The Enum plugin checks string values of Keys by comparing it against a list of valid values.
+
+## Usage ##
+
+The plugin checks every Key in the Keyset for the Metakey `check/enum` containing a list with the syntax `['string1', 'string2', 'string3', ..., 'stringN']` and compares each value with the string value of the Key. If no match is found an error is returned.
```

Comment:
The [] isn't needed,i was trying to adapt the syntax of the specification file. Basically every alphanumeric string divided by an non-alphanumeric character is considered as a valid element right now, but i can change it to just use a comma as a valid seperator, values in <> to keys and not quoted values to plugin configured values ?